tests/expected/derive-invariant/invariant_fail/invariant_fail.rs

```diff
@@ -0,0 +1,29 @@
+// Copyright Kani Contributors
+// SPDX-License-Identifier: Apache-2.0 OR MIT
+
+//! Check that a verification failure is triggered when the derived `Invariant`
+//! method is checked but not satisfied.
+
+extern crate kani;
+use kani::Invariant;
+
+#[derive(kani::Arbitrary)]
+struct NotNegative(i32);
```

Comment:
Maybe add a comment here:

```suggestion
/// Incorrect usage of `Arbitrary` and `Invariant` types.
///
/// `Arbitrary` implementation should respect the type invariant.
#[derive(kani::Arbitrary)]
struct NotNegative(i32);
```

Reply:
Added manually in addition to the link to the issue.